decred/tests/integration/dcr/test_rpc_live.py

```diff
@@ -315,7 +320,7 @@ def test_Client(config):
             break
 
     else:
-        raise DecredError("did not find a suitable script to decode")
+        raise RuntimeError("did not find a suitable script to decode")
```

Comment:
This is not a domain error but a testing coherence one. Cannot use `AssertionError` either because it would be reported by pytest as a test failure rather than an error.